Source/Core/Core/GeckoCodeConfig.cpp

```diff
@@ -30,12 +30,15 @@ std::vector<GeckoCode> DownloadCodes(std::string gameid, bool* succeeded)
     break;
   }
 
-  std::string endpoint{"https://geckocodes.org/txt.php?txt=" + gameid};
+  std::string endpoint{"https://www.geckocodes.org/txt.php?txt=" + gameid};
   Common::HttpRequest http;
 
   // Circumvent high-tech DDOS protection
   http.SetCookies("challenge=BitMitigate.com;");
 
+  // The server always redirects once to the same location.
+  http.FollowRedirects(1);
```

Comment:
Might as well make this 3 or so for future proofing.

Reply:
this is currently implemented in a GUI blocking way, so if the server starts redirecting a lot I'd prefer we know about it